install/kubernetes/helm/istio/values.yaml

```diff
@@ -336,49 +344,32 @@ security:
 #
 # addons configuration
 #
+telemetry-gateway:
+  gatewayName: ingressgateway
+  grafanaEnabled: false
+  prometheusEnabled: false
+
 grafana:
   enabled: false
   replicaCount: 1
   image: grafana
+  security:
+    enabled: false
+    adminUser: admin
+    adminPass: admin
```

Comment:
adminPassword here?  since you used `.Values.security.adminPassword` earlier.

Reply:
Great catch!